### PR TITLE
Update 0.133 release notes

### PR DESCRIPTION
**Pull request by raghavsethi**

### CODE REVIEW

**Comment by martint** [comment]
These should go in 0.134. We released 0.133 today

**Comment by electrum** [comment]
Mate both these changes in 0.133? If so, we can update the website with this.

**Comment by raghavsethi** [comment]
Sorry, sticky headers/sort was in 0.133 and the other is in 0.134. I'll split them up.

File: presto-docs/src/main/sphinx/release/release-0.133.rst [hunk 1]
```diff
@@ -16,3 +16,6 @@ General Changes
   ``dictionary_aggregation`` session property.
 * Fix race which could cause queries to fail when using :func:`concat` on
   :ref:`array_type`, or when enabling ``columnar_processing_dictionary``.
+* Add cumulative memory statistics tracking, and expose the stat in the web interface.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.133.rst]: Remove comma before "and"

File: presto-docs/src/main/sphinx/release/release-0.133.rst [hunk 2]
```diff
@@ -16,3 +16,5 @@ General Changes
   ``dictionary_aggregation`` session property.
 * Fix race which could cause queries to fail when using :func:`concat` on
   :ref:`array_type`, or when enabling ``columnar_processing_dictionary``.
+* Add sticky headers and the ability to sort the tasks table on the query page
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.133.rst]: Maybe flip the sentence order:

"Update query page in the web interface to have sticky headers and the ability to sort the tasks table."

**Comment by electrum** [comment]
Looks good

**Pull request merged**